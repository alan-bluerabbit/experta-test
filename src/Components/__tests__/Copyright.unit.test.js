import React from 'react';
import renderer from 'react-test-renderer';
import Copyright from '../Copyright';

test('<Copyright />', () => {
    const tree = renderer
    .create(<Copyright />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
