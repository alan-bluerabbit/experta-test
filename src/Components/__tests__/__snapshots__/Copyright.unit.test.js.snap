// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Copyright /> 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
>
  Copyright Â© 
  <a
    className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorInherit"
    href="https://www.experta.com.ar"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    Experta Seguros
  </a>
   
  2020
  .
</p>
`;
